src/entity.js

```diff
@@ -0,0 +1,25 @@
+// You should create a new file for each entity object
```

Comment:
I am not sure, if we really need entity as a golang, because the functionality it self is different, not as good as golang entity.

Reply:
if we don't create an entity, the object can become so dynamic and doesn't have any validation.
Let say we want to use store an article, instead of using
```js
let article = {
  id: '123',
  title: 'article title'
  // other field
}
```
which someone can forget what other fields are, we can use a function
```js
let article = Article('123', 'article title', ....)
```
the person can know what other fields the article needs to be filled. We also can put a validation inside the function.

Reply:
There is a problem, when you don't insert fields completely.
```js
function Article(id, title, content, publisherID) {
  return {
    id,
    title,
    content,
    publisher: {
      id: publisherID
    }
  };
}

console.log(JSON.stringify(Article(1))); // output: {"id":1,"publisher":{}}
```
I think you should create a constant which store default fields to be assigned.
```js
const ARTICLE = {
    id: '',
    title: '',
    content: '',
    publisher: {
      id: ''
    }
};
```
and inside `Article` function, you must create a condition.
```js
const article = ARTICLE;
if (id) {
  article.id = id;
}
...
```

Reply:
or maybe simply try like this
```js
return {
    id: id ? id : '',
    title,
    content,
    publisher: {
      id: publisherID
    }
  };
```